controllers/azuremanagedcluster_controller_test.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2024 The Kubernetes Authors.
```

New files always get the current year for copyright, is my understanding.